src/precice_to_aste.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import subprocess
+import argparse, logging
+
+def run(cmd):
+    print("+ " + cmd)
+    subprocess.run(cmd, shell = True, check = True)
+    
+# Generate Aste files from preCICE VTK output for every timestep
+def main():    
+    args=parse_args()
+    files = []
+    files.append(args.SolverName + ".init.vtk")
+    
+    for i in range(1,args.n+1):
+        files.append(args.SolverName +".dt" + str(i) + ".vtk")
+    
```

Comment:
Please check if the above all `files` actually exist and tell the user which ones are missing.
This makes debugging easier.